odsApi_versioned_docs/version-7.3/how-to-guides/how-to-add-a-postgre-role-on-a-custom-configured-postgresql-server.md

```diff
@@ -0,0 +1,39 @@
+# **How to: Add a `postgres` Role on a custom-configured PostgreSQL Server**
```

Comment:
Capitalize the `T` in `How to` to be consistent with the rest of the guides.
Also, remove the enclosing asterisks (**) since they get rendered in the navbar: 

![image](https://github.com/user-attachments/assets/ee1fca68-636b-4fb0-9363-d4c77ece5a1c)